Testscripts/Linux/verify_vmbus_interrupts.sh

```diff
@@ -57,16 +62,18 @@ function verify_vmbus_interrupts() {
                 fi
             done
         fi
-    done < "/proc/interrupts"
+    done < interrupts
 
-    if [ $nonCPU0inter -ne "$cpu_count" ]; then
-        LogErr "Not all CPU cores are processing VMBUS interrupts!"
+    if [ $nonCPU0inter -eq 0 ]; then
+        LogMsg "Total CPU counts: $cpu_count"
+        LogMsg "The number of CPU counts using interrupts: $nonCPU0inter"
+        LogErr "None of CPU cores are processing VMBUS interrupts!"
         SetTestStateFailed
-        exit 0
+    else
+        UpdateSummary "All {$cpu_count} CPU cores are processing interrupts."
+        SetTestStateCompleted
     fi
-
-    UpdateSummary "All {$cpu_count} CPU cores are processing interrupts."
-    SetTestStateCompleted
 }
 
-verify_vmbus_interrupts
\ No newline at end of file
+verify_vmbus_interrupts
+exit 0
```

Comment:
Is there any specific reason behind "exit 0" here? What happens if we don't put this here?

Reply:
No specific reason, but exit 0 is safely terminated the execution in bash.